### DISCUSSION TITLE
Client-only mode

### DISCUSSION THREAD

**Discussion by crabdancing** [rustdesk/rustdesk#5390] [Feature Request]
To minimize attack surface for high-security management systems, it would be ideal to not have the server side enabled/operating at all. 

1) Feature flags could be added, to make a hard-compiled client-only version available.
2) Alternatively, a flag could be set during runtime that would disable the username/password for the local system, denying all access.

**Comment by xlionjuan** [comment]
No client  only feature right now, but you can set it to reject all connection by default

**Reply by crabdancing** [reply]
How would I do that?
